.github/actions/gradle-build-action/action.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: "gradle-build"
+description: "Setup a specific JDK version and run gradle build"
+
+inputs:
+  jdk-version:
+    description: "OpenJDK version"
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - id: Setup JDK ${{ inputs.jdk-version }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ inputs.jdk-version }}
+          distribution: 'oracle'
```

Comment:
Oracle? Should we be using OpenJDK instead?

Reply:
`temurin` is the Eclipse Foundation's OpenJDK prebuilt package.

Reply:
thank you! let me switch that